### PR TITLE
Dockerization

### PR DESCRIPTION
**Pull request by sidorovdmitry**
Initial Docker support. Need some additional work for production deployment and container registry. But for now it's ready for local development.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,44 +2,53 @@
 Acceptor server which takes specific data from edx-platform and visualizes the data on the world map.
 
 # installations
```
Comment (sidorovdmitry) [inline README.md]: It's all from https://docs.docker.com/

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+version: '2'
+services:
+  acceptor:
+    container_name: acceptor
+    env_file: ./envs/pg_prod.env
+    build:
+      context: ./web/
+      dockerfile: Dockerfile
+    command: "gunicorn acceptor.wsgi -b 0.0.0.0:8000"
```
Comment (sidorovdmitry) [inline docker-compose.yml]: It's for production, not for local development.

File: .dockerignore [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.DS_Store
+.dockerignore
+.git/
+.gitignore
+README.md
+*.pyc
+__pycache__
```
Comment (sidorovdmitry) [inline .dockerignore]: Done

File: .travis.yml [hunk 4]
```diff
@@ -3,13 +3,19 @@ sudo: false
...
+
 cache:
   pip: true
 
 before_install:
 - pip install -r web/requirements.txt
 - pip install -r web/requirements_dev.txt
 
+before_script:
+- psql -c 'create database travis_ci_test;' -U postgres
```
Comment (sidorovdmitry) [inline .travis.yml]: Are you sure about database name?

Comment (DmytryiStriletskyi) [inline .travis.yml]: It is just test, that Travis work with this command.

File: config/database.yml.travis [hunk 5]
```diff
@@ -0,0 +1,7 @@
+test:
+  adapter: postgresql
+  database: acceptor
+  username: postgres
+  host: postgres
+  password: password
+  encoding: utf8
```
Comment (auraz) [inline config/database.yml.travis]: Add new line at the end of file.

File: config/database.yml.travis [hunk 6]
```diff
@@ -0,0 +1,7 @@
+test:
+  adapter: postgresql
+  database: acceptor
+  username: postgres
+  host: postgres
```
Comment (auraz) [inline config/database.yml.travis]: Why you need this setting?

File: config/database.yml.travis [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+  password: password
```
Comment (auraz) [inline config/database.yml.travis]: maybe you do not need this file at all.

**Comment by auraz** [comment]
please squash commits before merge

**Pull request merged**